stream_alert_cli/helpers.py

```diff
@@ -30,10 +30,56 @@
 from botocore.exceptions import ClientError
 from moto import mock_cloudwatch, mock_kms, mock_kinesis, mock_lambda, mock_s3
 
+from app_integrations.config import AWS_RATE_RE
 from stream_alert_cli.logger import LOGGER_CLI
 from stream_alert_cli.terraform._common import enabled_firehose_logs
 
 
+class ParseError(Exception):
+    """Custom expection rasied when argument parser has error"""
+
+def validate_scheduled_interval(val):
```

Comment:
As discussed offline, can we actually do away with the helpers and just duplicate this code (sigh) in the arg parsers? I don't love that this will actually raise an exception and print the traceback, instead of acting like the normal argparse error that prints the usage and the error (without printing a traceback). I'm sorry :(

Reply:
:( Yeah, I will make change. I am agree that it is ugly to see the traceback when there is argument parse error. So we have to make a compromise that to use clean build-in argument error handling (have to duplicate code) over custom exception which will print out traceback.